routers/web/repo/editor.go

```diff
@@ -40,37 +40,47 @@ const (
 	frmCommitChoiceNewBranch string = "commit-to-new-branch"
 )
 
-func canCreateBasePullRequest(ctx *context.Context) bool {
-	baseRepo := ctx.Repo.Repository.BaseRepo
+func canCreateBasePullRequest(ctx *context.Context, editRepo *repo_model.Repository) bool {
+	baseRepo := editRepo.BaseRepo
 	return baseRepo != nil && baseRepo.UnitEnabled(ctx, unit.TypePullRequests)
 }
 
-func renderCommitRights(ctx *context.Context) bool {
-	canCommitToBranch, err := ctx.Repo.CanCommitToBranch(ctx, ctx.Doer)
-	if err != nil {
-		log.Error("CanCommitToBranch: %v", err)
+func renderCommitRights(ctx *context.Context, editRepo *repo_model.Repository) bool {
+	if editRepo == ctx.Repo.Repository {
```

Comment:
```suggestion
	if editRepo.ID == ctx.Repo.Repository.ID {
```

Reply:
Will change it here and in two other places.